api/tests/acceptance/application/airtable-proxy/airtable-proxy-controller-skill_test.js

```diff
@@ -10,7 +10,7 @@ import {
 } from '../../../test-helper.js';
 import { createServer } from '../../../../server.js';
 
-describe('Acceptance | Controller | airtable-proxy-controller | create skill translations', () => {
+describe.skip('Acceptance | Controller | airtable-proxy-controller | create skill translations', () => {
```

Comment:
```suggestion
describe('Acceptance | Controller | airtable-proxy-controller | create skill translations', () => {
```
C'est normal qu'on ait gardé ces tests en skip ?